lib/src/components/NxTransferList/NxTransferList.tsx

```diff
@@ -38,11 +38,13 @@ function TransferListItem<T extends string | number>(props: TransferListItemProp
   );
 }
 
+TransferListItem.propTypes = transferListItemPropTypes;
+
 const defaultItemsCountFormatter = (kind: string) => (n: number) => `${n} item${n === 1 ? '' : 's'} ${kind}`,
     defaultAvailableItemsCountFormatter = defaultItemsCountFormatter('available'),
     defaultSelectedItemsCountFormatter = defaultItemsCountFormatter('transferred');
 
-export default function NxTransferList<T extends string | number>(props: Props<T>) {
+const NxTransferList = <T extends string | number>(props: Props<T>) => {
```

Comment:
Why this change?

Reply:
> Why this change?

Ops, I changed it out of habit when I removed the `export default`, will change it back to use function.

Reply:
> Why this change?
This was changed back in this commit:
0f0c1c1799991ec9d4c9e64eca29b9c86c4061c0